compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1184,3 +1184,12 @@ error.invalid.package.name.qualifier=\
 
 error.identifier.literal.only.supports.alphanumerics=\
   identifier literal only supports alphanumeric characters
+
+error.incompatible.mapping.constructor.expression=\
+  incompatible mapping constructor expression for type ''{0}''
+
+error.mapping.constructor.compatible.type.not.found =\
+  a type compatible with mapping constructor expressions not found in type ''{0}''
+
+error.cannot.infer.types.for.tuple.binding =\
+  invalid expression: types cannot be inferred for ''{0}'' 
```

Comment:
The error message is not related to a tuple, but the error key says tuple binding.

Reply:
It seems like no test case is added for this particular case.

Reply:
This error is only for tuples (lists). Will modify the error message to make it clear.